src/Framework/Microsoft.Build.Framework.csproj

```diff
@@ -170,5 +170,11 @@
   <ItemGroup>
     <Folder Include="Properties\" />
   </ItemGroup>
+  <ItemGroup>
+    <FilesToSign Include="$(OutDir)\$(AssemblyName).dll">
```

Comment:
For the file name, can you try `$(AssemblyName)$(TargetExt)` ?  For all of these